v2/skins.js

```diff
@@ -80,6 +84,7 @@
 //   * "ShowInWardrobe"
 //   * "HideIfLocked" -- doesn't show up in the wardrobe if not unlocked
 //   * "NoCost" -- doesn't cost transmutation charges to apply
+//   * "OverrideRarity" -- skin overrides item rarity when applied
```

Comment:
Could this be clarified a bit more? I don't understand what it is supposed to do.